iroh-resolver/src/dns_resolver.rs

```diff
@@ -0,0 +1,157 @@
+use crate::resolver::Path;
+use anyhow::Result;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use std::net::{IpAddr, Ipv4Addr};
+use trust_dns_resolver::config::{NameServerConfigGroup, ResolverConfig, ResolverOpts};
+use trust_dns_resolver::{AsyncResolver, TokioAsyncResolver};
```

Comment:
can you split imports into `std`, `other crates` and `crate::` please?

Reply:
Wow, I relied upon rustfmt. May it be worth to add `rustfmt.toml` with `group_imports = "StdExternalCrate"` in the root? It will require launching `fmt` through `cargo +nightly fmt` but relieve from manual playing with order.

Reply:
Yeah I am always torn about that. It works pretty well if you set it up manually splitting them by lines (then rustfmt keeps them sorted at least)

Reply:
looks like it is stable now: https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#group_imports

Reply:
made a tracking issue for now https://github.com/n0-computer/iroh/issues/550

Reply:
Done